integration/spark2/src/test/scala/org/apache/spark/carbondata/TestStreamingTableOperation.scala

```diff
@@ -775,6 +776,8 @@ class TestStreamingTableOperation extends QueryTest with BeforeAndAfterAll {
             .option("bad_records_action", badRecordAction)
             .option("dbName", tableIdentifier.database.get)
             .option("tableName", tableIdentifier.table)
+            .option(CarbonStreamParser.CARBON_STREAM_PARSER,
+              "org.apache.carbondata.streaming.parser.CSVStreamParserImp")
```

Comment:
Better to use default parser, 
Can you modify it, like CarbonStructuredStreamingExample.scala

Reply:
OK, I will modify it later.

Reply:
After PR#1962 is merged, I will modify it , otherwise there is error when write complex type data.